Debounce/DebouncerBase.cs

```diff
@@ -0,0 +1,412 @@
+﻿// SPDX-FileCopyrightText: 2021 Frans van Dorsselaer
+//
+// SPDX-License-Identifier: MIT
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Dorssel.Utilities.Generic;
+
+/// <summary>
+/// This class implements the <see cref="IDebouncerBase{TEventArgs}"/> interface.
+/// </summary>
+public abstract class DebouncerBase<TEventArgs>
+    : IDebouncerBase<TEventArgs>
+    , IDisposable
+    where TEventArgs : DebouncedEventArgs
+{
+    private protected DebouncerBase()
+    {
+        Timer = new Timer(OnTimer, null, Timeout.InfiniteTimeSpan, Timeout.InfiniteTimeSpan);
+    }
+
+    /// <summary>
+    /// 
```

Comment:
Missing documentation. Or to be removed...

Reply:
Done